<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Ames Grisanti</author>
 <class>Lepton</class>
 <widget class="QWidget" name="Lepton">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>890</width>
    <height>560</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>890</width>
    <height>560</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>890</width>
    <height>560</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Lepton</string>
  </property>
  <widget class="QWidget" name="Image" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>640</width>
     <height>480</height>
    </rect>
   </property>
  </widget>
  <widget class="QFrame" name="Buttons">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>500</y>
     <width>181</width>
     <height>50</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="UpdateFFC">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>161</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Update FFC</string>
    </property>
   </widget>
  </widget>
  <widget class="QPlainTextEdit" name="FLIR_SN">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>10</y>
     <width>71</width>
     <height>30</height>
    </rect>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="Uptime">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>50</y>
     <width>91</width>
     <height>30</height>
    </rect>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>10</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>S/N</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>50</y>
     <width>67</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Up Time</string>
   </property>
  </widget>
  <widget class="QFrame" name="FileButtons">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>500</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="SaveButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="VideoLive">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>200</y>
     <width>181</width>
     <height>61</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QGroupBox" name="VideoFreeze">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>0</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>Video Freeze</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QRadioButton" name="FreezeOn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>51</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>On</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="FreezeOff">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>30</y>
       <width>61</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Off</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <zorder>VideoFreeze</zorder>
   <zorder>Image</zorder>
  </widget>
  <widget class="QFrame" name="VideoPolarity">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>380</y>
     <width>181</width>
     <height>91</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QGroupBox" name="Video">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>0</y>
      <width>121</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>Video Polarity</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QRadioButton" name="WhiteHot">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>101</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>White Hot</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="BlackHot">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>101</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Black Hot</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QFrame" name="VideoAGCCont">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>260</y>
     <width>181</width>
     <height>61</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QGroupBox" name="VideoAGC">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>0</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>Video AGC</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QRadioButton" name="AGCOn">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>51</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>On</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="AGCOff">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>30</y>
       <width>61</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Off</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QFrame" name="SceneBasedNUC">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>320</y>
     <width>181</width>
     <height>61</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QGroupBox" name="SBNUC">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>Scene Based NUC</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QRadioButton" name="SBNUCOn">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>51</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>On</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="SBNUCOff">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>30</y>
       <width>61</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Off</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QFrame" name="CameraTemp">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>90</y>
     <width>181</width>
     <height>111</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QGroupBox" name="CameraTemperature">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>151</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>CameraTemperature</string>
    </property>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>FPA</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="FPATemp">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>30</y>
       <width>51</width>
       <height>30</height>
      </rect>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Case</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="AuxTemp">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>70</y>
       <width>51</width>
       <height>30</height>
      </rect>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>30</y>
       <width>16</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>C</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>70</y>
       <width>16</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>C</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
